src/Kunstmaan/AdminBundle/Twig/MultiDomainAdminTwigExtension.php

```diff
@@ -25,10 +28,10 @@ public function __construct(DomainConfigurationInterface $domainConfiguration)
     public function getFunctions()
     {
         return array(
```

Comment:
```array``` => ```[]``` :D

Reply:
The minimal version of branch 4.0 is PHP5.6 so PHP5.3 array notation is not necessary.

Reply:
replaced!